src/data/baseline-data.js

```diff
@@ -10,7 +10,7 @@ export const BASELINE_LOW = 5;
 export const BASELINE_FALSE = 0;
 
 export const properties = new Map([
-	["accent-color", "0:"],
+	["accent-color", "10:2022"],
```

Comment:
(non-blocking) FYI the failing tests are due to https://github.com/web-platform-dx/web-features/issues/2880. I'd expect the status for `accent-color` to go back to "false" once that's resolved.

Reply:
The manual override should no longer be necessary after updating to [web-features@2.35.1](https://github.com/web-platform-dx/web-features/releases/tag/v2.35.1) 👍